litmus-portal/frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "Hub for LitmusChaos",
+	"short_name": "ChaosHub",
```

Comment:
Why the name is chaosHub here?

Reply:
will change this name...also kept some code in the redux and models folder from chaoshub to provide some reference on building the actual redux workflows rather than keeping an empty folder.